app/controllers/admin/financial_transaction_classes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Admin::FinancialTransactionClassesController < Admin::BaseController
+  inherit_resources
+
+  def new
+    @financial_transaction_class = FinancialTransactionClass.new(params[:financial_transaction_class])
+    render :layout => false
```

Comment:
I would like to transition to new-style options `layout: false`, any objections?